packages/invoices/src/utils.ts

```diff
@@ -133,3 +134,14 @@ export const inboundLinesToSummaryItems = (
 export const canDeleteInvoice = (invoice: OutboundRowFragment): boolean =>
   invoice.status === InvoiceNodeStatus.New ||
   invoice.status === InvoiceNodeStatus.Allocated;
+
+export const isA = {
```

Comment:
🤔 Got a better idea? 😁 

I was thinking of also adding `updateStockOutLine` etc for the conditions in the upsert api fn 🤔

Reply:
I liked it - already looks super handy